builtin/common/misc_helpers.lua

```diff
@@ -238,6 +238,50 @@ function math.sign(x, tolerance)
 end
 
 --------------------------------------------------------------------------------
+-- Video enums and packing function
+
+-- E_BLEND_FACTOR
+core.ebf = {
```

Comment:
This should be an `EnumString` like [this](https://github.com/minetest/minetest/blob/1980d9ea31e969c5b604f6ee01693cbcfc2c795a/src/script/lua_api/l_object.cpp#L34).

Please, let's not add more items to #3980.